javascript-functions/main.js

```diff
@@ -0,0 +1,27 @@
+const convertMinutesToSeconds = (minutes) => Math.pow(10, 3) * 6;
+
+console.log(convertMinutesToSeconds(3));
+
+const greet = (name) => name;
+
+console.log(`Hey ${greet('Jack')}`);
+
+const getArea = (width, height) => {
+  return 0.5 * width * height;
+};
+
+console.log(getArea(3, 5));
+
+// getFirstName function
+const getFirstName = (person) => person.firstName;
+
+console.log(getFirstName({ firstName: 'Jack', lastName: 'Almadjian' }));
+console.log(getFirstName());
+
+const getLastElement = (
+  array = ['propane', 'and', 'propane', 'accessories']
+) => {
+  return array[array.length - 1];
+};
+
+console.log(getLastElement());
```

Comment:
Still one more change that hasn't been addressed!

You should be creating the array outside the function and passing it into your function call on line 27. You shouldn't be setting the values of the array inside the parameter.